budget-frontend/src/components/Register/Register.tsx

```diff
@@ -0,0 +1,111 @@
+import React from 'react';
+import {
+    Button,
+    Form,
+    Input,
+} from 'antd';
+
+interface DataNodeType {
+    value: string;
+    label: string;
+    children?: DataNodeType[];
+}
+
+const formItemLayout = {
+    labelCol: {
+        xs: { span: 24 },
+        sm: { span: 8 },
+    },
+    wrapperCol: {
+        xs: { span: 24 },
+        sm: { span: 16 },
+    },
+};
+
+const tailFormItemLayout = {
+    wrapperCol: {
+        xs: { span: 24, offset: 0 },
+        sm: { span: 16, offset: 4 },
+        lg: { span: 16, offset: 4 },
+    },
+};
+
+export const Register: React.FC = () => {
+    const [form] = Form.useForm();
+
+    const onFinish = (values: any) => {
```

Comment:
let's try to avoid using any as much as it's possible.
In this case we can create interface and use it for example ->

```ts
interface RegisterFormValues {
    email: string;
    password: string;
    confirm: string;
}
```